include/z64audio.h

```diff
@@ -205,7 +205,7 @@ typedef s16 AdpcmBookData[];
 
 typedef struct AdpcmBook {
     /* 0x00 */ AdpcmBookHeader header;
-    /* 0x08 */ AdpcmBookData book; // size 8 * order * numPredictors. 8-byte aligned
+    /* 0x08 */ s16 book[1]; // size 8 * order * numPredictors. 8-byte aligned
```

Comment:
I feel like it might be less confusing if `AdpcmBookData` were removed altogether and sfc be modified to output s16[] instead
`"NO_REORDER SECTION_DATA AdpcmBookData SF%d_%s_BOOK_DATA = {"                    "\n",`
to
`"NO_REORDER SECTION_DATA s16 SF%d_%s_BOOK_DATA[] = {"                            "\n",`

Reply:
Done, I'll admit I'm reluctant to touch the audio tools because you seem pretty insistent on keeping it in sync with MM and I don't really want to have to sync them myself

Reply:
I can sync this with MM, unless anyone else reading wants to go ahead and get to it before me